### PR TITLE
Update to v1.0.5

### PR DESCRIPTION
**Pull request by lexming**
This might seem as another huge PR, but it is not. Code is mostly moving around, but the real changes are small.

The big issue is that Python 3.6 does not support the parallelized execution done in `vsc-accounting` as it is currently done. There is a weird bug/non-bug (it's not clear, see https://bugs.python.org/issue29423) that only affects v3.6 and limits the functions that can be pickled by `ProcessPoolExecutioner` to be top-level functions. This limitation does **not** exists in Python v3.5, v3.7 or v3.8.

Therefore, I moved the code around to be compliant with this restriction. Nonetheless, the good part is that the code in this PR not only works in Python v3.6, but also in Python v3.8. So there will be no need to change this again.
* First change is to move the `ProcessPoolExecutioner` out of the classes into its own module. See https://github.com/sisc-hpc/vsc-accounting-brussel/commit/05702e685737f5a1ccba0cead1244aa9f440274a
* Second change is move all functions being parallelized out of their classes, which is basically copy pasting and removing references to `self`
  * this is done for `counters` in https://github.com/sisc-hpc/vsc-accounting-brussel/commit/040bf0fcef342c4075de3c9ff2bdad08bb9c8099
  * and for `userdb` in https://github.com/sisc-hpc/vsc-accounting-brussel/commit/c1239a1b78a649352f1d5ba45c376cf50528cdce

That's the bulk of the PR. Apart from that, there a few small improvements
* data files will be searched in `/etc` if defined by file name, see https://github.com/sisc-hpc/vsc-accounting-brussel/commit/8897219126b4d30576b66fb14d679151d5fe89c6
* added option to force the re-installation of config and data files from their source location into the user's directory, this is useful in case of updates of those files, see https://github.com/sisc-hpc/vsc-accounting-brussel/commit/3af3b5e2b482a23bb715a390bf8fb69046d88d7d
* updates to the README in https://github.com/sisc-hpc/vsc-accounting-brussel/commit/4d59eb0dbf3a39bce1befeac48c66c2a936b6261

I already tested this version in `cerberus` and it worked well.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -56,11 +56,14 @@
         # needed to process accounting data
         'numpy >= 1.14.0',
         'pandas >= 0.23.0',
```
Comment (wpoely86) [inline setup.py:58]: drop this line if you already have it below?

File: setup.py [hunk 2]
```diff
@@ -56,11 +56,14 @@
...
+        'pandas >= 0.23.0, < 1.0.0',
```
Comment (wpoely86) [inline setup.py:59]: can you add a comment on why the upper bound is needed?

File: setup.py (hunk 1 above)
Comment (lexming) [reply setup.py]: right! :+1:

**Review by wpoely86 — APPROVED** [review]

File: setup.py (hunk 2 above)
Comment (lexming) [reply setup.py]: comment added in https://github.com/sisc-hpc/vsc-accounting-brussel/pull/2/commits/0e6b8be9e5067f77ad7864527d26e2d23ba63b88

**Pull request merged**